app/resources/v1/ont/flowcell_resource.rb

```diff
@@ -2,25 +2,25 @@
 
 module V1
   module Ont
-    # @todo This documentation does not yet include a detailed description of what this resource represents.
-    # @todo This documentation does not yet include detailed descriptions for relationships, attributes and filters.
-    # @todo This documentation does not yet include any example usage of the API via cURL or similar.
+    # Provides a JSON:API representation of {Ont::Flowcell}.
+    #
+    ## Primary relationships:
+    # * pool {V1::PoolResource} - The pool loaded onto the flowcell.
     #
     # @note Access this resource via the `/v1/ont/flowcells/` endpoint.
     #
-    # Provides a JSON:API representation of {Ont::Flowcell}.
+    # @example
+    #   curl -X GET http://localhost:3100/v1/ont/flowcells
+    #   curl -X GET "http://localhost:3100/v1/ont/flowcells/1"
     #
-    # For more information about JSON:API see the [JSON:API Specifications](https://jsonapi.org/format/)
-    # or look at the [JSONAPI::Resources](http://jsonapi-resources.com/) package
-    # for the service implementation of the JSON:API standard.
     class FlowcellResource < JSONAPI::Resource
       model_name 'Ont::Flowcell'
 
-      # @!attribute [rw] flowcell_id
+      # @!attribute [r] flowcell_id
       #   @return [Integer] the ID of the flowcell
-      # @!attribute [rw] position
+      # @!attribute [r] position
       #   @return [String] the position of the flowcell
-      # @!attribute [rw] ont_pool_id
+      # @!attribute [r] ont_pool_id
```

Comment:
If these are readonly do we need to add `readonly: true` to the attributes below?